examples/simple/main.tf

```diff
@@ -20,13 +20,13 @@ module "materialize_infrastructure" {
   single_nat_gateway   = true
 
   # EKS Configuration
-  cluster_version = "1.31"
-  # node_group_instance_types                = ["m6g.medium"]
+  cluster_version = "1.32"
   # TODO: Defaulting to a smaller instance type due to resource constraints
+  # node_group_instance_types                = ["r7g.xlarge"]
   node_group_instance_types                = ["r5.xlarge"]
```

Comment:
Should this line with `r5` go away since node ami type is being defaulted to  AL2023_ARM_64_STANDARD ?

Reply:
Oh yes, thanks for catching that Kay! 🙏